### PR TITLE
CAN-74: Manage Profile Frontend Design

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
CAN-74: Manage Profile Frontend Design

### CODE REVIEW

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    <Select defaultValue="private" className="select-after">
+      <Option value="private">Private</Option>
+      <Option value="public">Public</Option>
+    </Select>
+  );
+
+
+  return (
+    <section className={styles.profileInfo_wrapper}>
+      <Form
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: There are multiple `Form`, please split it into its own components

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      >
+        <div className={styles.section_one}>
+          <Row gutter={30}>
+            <Col md={12}>
+              <Form.Item
+                name="phoneNumber"
+                label={messages.labels.phoneNumber}
+                rules={[
+                  {
+                    required: true,
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Try to move validation into its own component or package

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+                <Input placeholder={messages.placeholders.phoneNumber} size="large" />
+              </Form.Item>
+            </Col>
+            <Col md={12}>
+              <Form.Item
+                name="mobileCarrier"
+                label={messages.labels.mobileCarrier}
+                rules={[
+                  {
+                    required: true,
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Move validation into a separate Component or package

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+              htmlType="submit"
+              className="ant-btn ant-btn-orange ant-btn-lg"
+            >
+              Verify
+            </Button>
+        </div>
+      </Form>
+
+
+      <Form
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: It must come from a component

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+        </Title>
+
+        <div className={styles.section_two}>
+          <Row gutter={30}>
+            <Col md={12}>
+              <Form.Item
+                name="first_name"
+                label={messages.labels.firstName}
+                rules={[
+                  {
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Move validation into a separate Component or package

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+                ]}
+              >
+
+                <Input addonAfter={selectAfter} placeholder={messages.placeholders.firstName} size="large" />
+              </Form.Item>
+              <Form.Item
+                name="last_name"
+                label={messages.labels.lastName}
+                rules={[
+                  {
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Move validation into a separate Component or package

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+                </Radio.Group>
+
+              </Form.Item>
+
+            </Col>
+            <Col md={12}>
+              <Form.Item
+                name="middle_name"
+                label={messages.labels.middleName}
+                rules={[
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Move validation into a separate Component or package

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+                    message: messages.validations.middleName,
+                  },
+                ]}
+              >
+                <Input addonAfter={selectAfter} placeholder={messages.placeholders.middleName} size="large" />
+              </Form.Item>
+              <Form.Item
+                name="email"
+                label={messages.labels.email}
+                rules={[
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Move validation into a separate Component or package

**Review by akuks — CHANGES_REQUESTED** [review]
Please go through the comments. 
Thank you

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 3 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 4 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 5 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 6 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 7 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 8 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: Done

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/VerifyMobileNumber.tsx [hunk 9]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Row, Col, Form, Input, Button } from "antd";
+import styles from "./ProfileInfo.module.scss";
+import messages from "../../../../messages";
+
+function VerifyMobileNumber() {
```
Comment (akuks) [inline src/components/ComponentPages/ProfileInfo/ProfileInfoUI/VerifyMobileNumber.tsx]: I think the filename `VerifyMobileNumberForm` is more appropriate.  And it should be under the `Form` folder

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx (hunk 4 above)
Comment (akuks) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx]: You missed the put the profileInfo form into a different component

**Review by akuks — CHANGES_REQUESTED** [review]
I think most of the things are okay. Only change, that I requested

File: src/components/ComponentPages/ProfileInfo/ProfileInfoUI/VerifyMobileNumber.tsx (hunk 9 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ProfileInfo/ProfileInfoUI/VerifyMobileNumber.tsx]: Done

**Review by akuks — APPROVED** [review]

**Pull request merged**